### PR TITLE
Support OTP apps in "Go to package"

### LINKED ISSUE #1912: Better integrate Erlang/Elixir
* Popovers should work with Erlang too
* The "Go to Hexdocs" popup should be aware of Erlang apps

### PR DESCRIPTION
**Pull request by wojtekmach**
Closes #1912.

image

### CODE REVIEW

File: assets/css/quick-switch.css [hunk 1]
```diff
@@ -1,3 +1,7 @@
+#quick-switch-modal-description {
+  padding-bottom: 8px;
+}
+
```
Comment (josevalim) [inline assets/css/quick-switch.css:4]:
```suggestion
```

File: assets/js/handlebars/templates/quick-switch-modal-body.handlebars [hunk 2]
```diff
@@ -1,5 +1,10 @@
+<div id="quick-switch-modal-description">
+  Elixir and Erlang/OTP applications are also supported.
+</div>
```
Comment (josevalim) [inline assets/js/handlebars/templates/quick-switch-modal-body.handlebars:3]:
```suggestion
```

**Review by josevalim — APPROVED** [review]
Remove the hint and ship it!

File: assets/js/handlebars/templates/quick-switch-modal-body.handlebars [hunk 3]
```diff
@@ -1,5 +1,7 @@
 <div id="quick-switch-modal-body">
-  <i class="ri-search-2-line" aria-hidden="true"></i>
-  <input type="text" id="quick-switch-input" class="search-input" placeholder="Jump to..." autocomplete="off" spellcheck="false">
-  <div id="quick-switch-results"></div>
+  <div>
+    <i class="ri-search-2-line" aria-hidden="true"></i>
+    <input type="text" id="quick-switch-input" class="search-input" placeholder="Jump to..." autocomplete="off" spellcheck="false">
+    <div id="quick-switch-results"></div>
+  </div>
```
Comment (wojtekmach) [inline assets/js/handlebars/templates/quick-switch-modal-body.handlebars:6]:
```suggestion
  <i class="ri-search-2-line" aria-hidden="true"></i>
  <input type="text" id="quick-switch-input" class="search-input" placeholder="Jump to..." autocomplete="off" spellcheck="false">
  <div id="quick-switch-results"></div>
```

**Pull request merged**